components/layout_2020/flow/mod.rs

```diff
@@ -1621,24 +1621,24 @@ impl PlacementState {
                 }
                 fragment.content_rect.start_corner.block +=
                     Length::from(self.current_margin.solve()) +
-                        self.current_block_direction_position;
+                        self.current_block_direction_position.into();
```

Comment:
Nit: you can do the sum before converting:

```rs
                fragment.content_rect.start_corner.block +=
                    (self.current_margin.solve() + self.current_block_direction_position).into();
```

Reply:
Sure. I will apply the changes.